docs/csharp/how-to/compare-strings.md

```diff
@@ -55,12 +55,13 @@ When performing a case-insensitive ordinal comparison, these methods use the cas
 
 ## Linguistic comparisons
 
-Strings can also be ordered using linguistic rules for the current culture.
+Unless overridden, many string comparison methods (such as <xref:System.String.StartsWith%2A?displayProperty=nameWithType>) default to using linguistic rules for the _current culture_ to order their inputs.
```

Comment:
Let's not use "overridden", because that has a clear meaning that doesn't apply to `StartsWith` because it's not virtual:

```suggestion
Many string comparison methods (such as <xref:System.String.StartsWith%2A?displayProperty=nameWithType>) use linguistic rules for the _current culture_ to order their inputs.
```

Reply:
Good point. I updated my suggestion. It could be just me, but I put the "by default" after "for the current culture". I don't have strong feelings either way.